modules/auxiliary/gather/rails_doubletap_file_read.rb

```diff
@@ -0,0 +1,104 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'        => 'Ruby On Rails File Content Disclosure (\'doubletap\')',
```

Comment:
```suggestion
        'Name'        => "Ruby On Rails File Content Disclosure ('doubletap')",
```

Also, add `doubletap` to the `AKA` references.